python/lsst/ap/association/diaPipe.py

```diff
@@ -112,23 +112,53 @@ def adjustQuantum(self, datasetRefMap: pipeBase.InputQuantizedConnection):
 
         Parameters
         ----------
-        datasetRefMap : `NamedKeyDict`
-            Mapping from dataset type to a `set` of
-            `lsst.daf.butler.DatasetRef` objects
+        inputs : `dict`
+            Dictionary whose keys are an input (regular or prerequisite)
+            connection name and whose values are a tuple of the connection
+            instance and a collection of associated `DatasetRef` objects.
+            The exact type of the nested collections is unspecified; it can be
+            assumed to be multi-pass iterable and support `len` and ``in``, but
+            it should not be mutated in place.  In contrast, the outer
+            dictionaries are guaranteed to be temporary copies that are true
+            `dict` instances, and hence may be modified and even returned; this
+            is especially useful for delegating to `super` (see notes below).
+        outputs : `Mapping`
+            Mapping of output datasets, with the same structure as ``inputs``.
```

Comment:
Did you intend for `inputs` to be `dict` but `outputs` (and everything else) to be `Mapping`?

Reply:
Oh, good catch. I did not, and that's a mistake down in `pipe_base`, too (should be `dict` throughout).  I'll fix it here on this branch, but fix up pipe_base on a different ticket in that package that I already have underway.